components/main_menu/elements/icon_button_common.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+interface Props {
+    iconGlyph: string;
+    size: string;
+    label: string;
+    disabled?: boolean;
+    additionalClass?: string;
+}
+
+const IconButtonCommon: React.FC<Props> = ({iconGlyph, size, label, disabled, additionalClass}: Props) => {
+    return (
+        <div
+            className='IconButton__wrapper'
+        >
+            <button
+                className={`IconButton IconButton___${size} ${additionalClass}`}
+                disabled={disabled}
+                tabIndex={-1}
+                aria-label={label}
```

Comment:
This will definitely be something we can add later, but I wonder how often we'll want to just directly translate this label. If so, adding in i18n support might be a good idea eventually.